docker/Dockerfile.rocm

```diff
@@ -114,6 +114,12 @@ COPY --from=export_vllm /examples ${COMMON_WORKDIR}/vllm/examples
 ENV RAY_EXPERIMENTAL_NOSET_ROCR_VISIBLE_DEVICES=1
 ENV TOKENIZERS_PARALLELISM=false
 
+# ENV that can improve safe tensor loading, and end-to-end time
+ENV SAFETENSORS_FAST_GPU=1
```

Comment:
> ENV that can improve safe tensor loading,


I didn't find this variable in the vllm repository. Could you remind me why it can improve loading time?

Reply:
This is for the safe tensor, see https://huggingface.co/docs/safetensors/en/speed for more details.